### PR TITLE
Hometask 3

### PR DESCRIPTION
**Pull request by rkreddyy**

### CODE REVIEW

File: TrainingApp/src/components/Input/Input.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import React, { useReducer, useEffect } from 'react';
+import { View, Text, TextInput } from 'react-native';
+import { INPUT_BLUR, INPUT_CHANGE } from '../../store/actions/actionTypes';
+import styles from './style'
+
+const inputReducer = (state, action) => {
```
Comment (sharad-epam) [inline TrainingApp/src/components/Input/Input.js:6]: Separate Reducer from Component

File: TrainingApp/src/screens/Auth/AuthScreen.js [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+                                    keyboardType="default"
+                                    secureTextEntry
+                                    required
+                                    minLength={5}
+                                    autoCapitalize="none"
+                                    errorText="Please choose a Zone."
+                                    onInputChange={inputChangeHandler}
+                                    initialValue=""
+                                /> */}
+
```
Comment (sharad-epam) [inline TrainingApp/src/screens/Auth/AuthScreen.js:265]: remove the commented code

File: TrainingApp/src/screens/Shop/ProductsList/ProductsList.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import { View, Text, TouchableHighlight } from 'react-native';
+
+import Styles from './styles';
+import { BaseStyles } from '../../../app.styles';
+import Product from '../Product';
+
+export default class ProductList extends Component {
+    viewAllGroupItems() { }
```
Comment (sharad-epam) [inline TrainingApp/src/screens/Shop/ProductsList/ProductsList.js:9]: remove this empty method

File: TrainingApp/src/screens/Shop/Shop.js [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+            { id: 'efvedvevewvr', name: 'Cloth', source: require('../../images/cloth.jpg') },
+            { id: 'sdfdscdscvdsfvsdfvsdf', name: 'Furniture', source: require('../../images/furniture.jpg') },
+        ];
+        this.currentGroup = this.groups[0];
+    }
+    openMenu() {
+        this.props.navigation.openDrawer();
+    }
+
+    openWishList() {}
```
Comment (sharad-epam) [inline TrainingApp/src/screens/Shop/Shop.js:56]: remove this empty method

File: TrainingApp/src/components/Input/Input.js (hunk 1 above)
Comment (rkreddyy) [reply TrainingApp/src/components/Input/Input.js:6]: Done as part of Home Task 4

File: TrainingApp/src/screens/Auth/AuthScreen.js (hunk 2 above)
Comment (rkreddyy) [reply TrainingApp/src/screens/Auth/AuthScreen.js:265]: Done as part of Home Task 4

File: TrainingApp/src/screens/Shop/Shop.js (hunk 4 above)
Comment (rkreddyy) [reply TrainingApp/src/screens/Shop/Shop.js:56]: Done as part of Home Task 4

**Pull request merged**